docs/UserGuide.md

```diff
@@ -79,6 +80,23 @@ Examples:
 * `add title:Birthday party type:event desc:Frank’s birthday party date:01-01-2020 18:00`
   Adds a task with title “Birthday party” , type event,
   description “Frank’s birthday party” , and date and time “01-01-2020 18:00” to PlaNus.
+  
+### Add a lesson: `lesson`
+
+Adds a lesson to PlaNus.
+
+Format: `lesson title:TITLE [desc:DESCRIPTION] day:DAY from:TIME to:TIME start:DATE end:DATE`
+
+* Adds multiple recurring tasks of  `type:lesson` to PlaNus, starting from the date specified in `start:DATE` to the
+ date specified in `end:DATE`, on the day specified in `day:DAY` from the time specified in `from:TIME` to the time
+ specified in `to:TIME`.
+* The format of day must be as follows (case-insensitive):
```

Comment:
Not sure if it is just me but i usually write thursday as 'thurs' in short from but it must be 'thu' for the input. Maybe can make it clearer?